code/modules/surgery/surgery.dm

```diff
@@ -101,7 +101,7 @@
 		var/obj/item/tool = user.get_active_held_item()
 		if(S.try_op(user, target, user.zone_selected, tool, src, try_to_fail))
 			return TRUE
-		if(tool?.item_flags & SURGICAL_TOOL) //Just because you used the wrong tool it doesn't mean you meant to whack the patient with it
+		if(tool && tool.item_flags & SURGICAL_TOOL) //Just because you used the wrong tool it doesn't mean you meant to whack the patient with it
```

Comment:
There's possibly more of this class of errors, will have to search for procs that return obj/FALSE.

Reply:
Seems like the kind of thing that should be linted for.